hack/tools/kapinger/servers/http.go

```diff
@@ -5,6 +5,7 @@ import (
 	"fmt"
 	"log"
 	"net/http"
+	_ "net/http/pprof"
```

Comment:
what does this do? Comment for future users?

Reply:
@huntergregory that's fairly common, but also not my most favorite thing. It uses `init()` to attach pprof handlers to the default `net/http` handler. Probably fine here, where it's for testing, but for production pprof handlers I usually like to explicitly attach those handlers to my routes. This is documented in the package docs here: https://pkg.go.dev/runtime/pprof .